src/views/primary-auth/PrimaryAuthForm.js

```diff
@@ -87,6 +89,7 @@ define([
         placeholder: Okta.loc('primaryauth.password.placeholder', 'login'),
         name: 'password',
         input: TextBox,
+        inputId: 'pass-signin',
```

Comment:
If we need consistency, we should change the fields in the old loginform to whatever we end up with here.